src/govuk/components/header/_index.scss

```diff
@@ -8,12 +8,18 @@
   $govuk-header-nav-item-border-color: #2e3133;
   $govuk-header-link-underline-thickness: 3px;
 
+  $govuk-header-rebrand-background: $govuk-brand-colour;
+  $govuk-header-rebrand-logo-bottom-margin: 2px;
+
   .govuk-header {
     @include govuk-font($size: 16);
```

Comment:
v5 includes a `$line-height: 1` here, which results in a slightly different line-height across versions - is that expected?

Reply:
@domoscargin where would I see the visual change of this, would it be in the header logo?

Reply:
@CharlotteDowns Yeah, if you compare two rebranded previews, there's a 20px line-height on the `<header>` element which the `.govuk-header__logo` inherits. In v5, this is 16px.

I was mostly looking at code, so I'm not actually sure if this results in a visual change - it's pretty darn close if so.

Reply:
Weirdly I have this PR showing 16px for the line height for `.govuk-header__logo` in the inspector 🤔.

<img width="1068" alt="Screenshot of line height computed as 16px in inspector" src="https://github.com/user-attachments/assets/dfe43cbb-1a6b-4c65-911c-dcabd9134a9c" />

Reply:
I think that's fine. 16px is the default font size for govuk-frontend's css and we only apply a font size for the GOV.UK text inside the span that contains that text.